PWGJE/EMCALJetTasks/AliAnalysisTaskPWGJEQA.h

```diff
@@ -80,9 +81,10 @@ class AliAnalysisTaskPWGJEQA : public AliAnalysisTaskEmcalJet {
   
   void                        UserCreateOutputObjects();
 
+  void                        SetUseAliEventCuts(Bool_t b)                         { fUseAliEventCuts    = b; }
```

Comment:
Don't you also need a setter for manual event cuts?

Reply:
yeah I forgot that, will correct it